sciLifeLab_utils/run_delivery.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import glob
+import re
+import subprocess
+
+def main(arg):
+    project = os.path.split(os.path.realpath(arg.source))[1]
+    move_from_path = "{}/*/results/".format(arg.source)
+    pathway = glob.glob(move_from_path)
+    pattern = re.compile(r"/[^/]+/[^/]+/[^/]+/[^/]+/[^/]+/([^/]+)/") #Matches the group betwen the 6th / /
+    samples = []
+    for path in pathway:
+        try:
+            samples.append(pattern.match(path).groups()[0])
+        except IndexError:
+            pass
+    
+    
+    for sample in samples:
+        dest = "/proj/{}/INBOX/{}/QC_analysis/{}".format(arg.uppnexid,project,sample)
+        try:
+             os.mkdir("/proj/{}/INBOX/{}/QC_analysis".format(arg.uppnexid,project))
+          
+        except OSError:
+            pass
```

Comment:
maybe here some warning is needed, in order to warn the user that the data will not be delivered as expected